src/buildRos2Type.ts

```diff
@@ -211,6 +211,24 @@ function normalizeType(type: string): string {
   return type;
 }
 export function buildRos2Type(lines: { line: string }[]): MessageDefinition {
+  // Special case: Empty message definition (e.g. std_msgs/Empty).
+  // Since IDL structs can't be empty, ROS adds a uint8 field in case the
+  // message definition is empty (e.g. std_msgs/Empty):
+  // https://github.com/ros2/rosidl/blob/e3b71ece/rosidl_adapter/rosidl_adapter/resource/struct.idl.em#L101
+  if (lines.length === 0) {
```

Comment:
This seems like a serialization time concern rather than something that would be in the `MessageDefinition` itself (which is more of an app description for the message fields). Is it possible to make this a rosmsg2-serialization thing rather than a message definition builder thing?

Reply:
What happens in ROS tooling when working with these empty messages? Does `ros topic echo` show the field? Or are they special-casing the `std_msgs/Empty` (or even all empty messages?).

In practice empty messages are rare so this is an edge case with limited risk and impact but figuring out what the expectations are for a ROS user and following those seems defensible.